modules/Roll Groups/rollGroups_details.php

```diff
@@ -49,9 +49,9 @@
         } else {
             $row = $result->fetch();
 
-            echo "<div class='trail'>";
-            echo "<div class='trailHead'><a href='".$_SESSION[$guid]['absoluteURL']."'>".__('Home')."</a> > <a href='".$_SESSION[$guid]['absoluteURL'].'/index.php?q=/modules/'.getModuleName($_GET['q']).'/'.getModuleEntry($_GET['q'], $connection2, $guid)."'>".__(getModuleName($_GET['q']))."</a> > <a href='".$_SESSION[$guid]['absoluteURL'].'/index.php?q=/modules/'.getModuleName($_GET['q'])."/rollGroups.php'>".__('View Roll Groups')."</a> > </div><div class='trailEnd'>".$row['name'].'</div>';
-            echo '</div>';
+            $page->breadcrumbs
+                ->add(__('View Roll Groups'), 'rollGroups.php')
+                ->add(__($row['name']));
```

Comment:
Looking good. In this case, `$row['name']` doesn't need wrapped in `__()` because it's a text value defined by the school itself.

Reply:
Well spotted Sandra. I've resolved now, and will merge and test.